### ISSUE TITLE
evaluate numerical optimization libraries

### ISSUE THREAD

**Issue by joacorapela** [joacorapela/svGPFA#2]
Pytorch's optim is not the best option for svGPFA, since we need to specify a learning rate for optimizations. These learning rates have an important impact on the quality of the optimization. Therefore, we should select a different optimization library (e.g., scipy optim). Done.

In a simple optimization test problem (e.g., rosenbrock) test if the specification of learning rates is problematic, evaluate the quality of different optimization libraries, as well as the runtime cost of autograd. Done.

Now test on another optimization test problem to understand the generality of the previous observations.
